configure.js

```diff
@@ -1,10 +1,17 @@
-var ipc = require('ipc')
 var Ractive = require('ractive')
 var page = require('page')
 var fs = require('fs')
+var Client = require('electron-rpc/client')
+var client = new Client()
```

Comment:
whoa didnt know you could do that, is it documented? (the `electron-rpc` module). im assuming its the same thing as ipc?